packages/javascript-sdk/src/index.ts

```diff
@@ -19,6 +19,7 @@ import AttributeInputCallback from './fr-auth/callbacks/attribute-input-callback
 import ChoiceCallback from './fr-auth/callbacks/choice-callback';
 import ConfirmationCallback from './fr-auth/callbacks/confirmation-callback';
 import DeviceProfileCallback from './fr-auth/callbacks/device-profile-callback';
+import { deviceClient } from './device-client';
```

Comment:
added it because if i dont i have to add a new entry point in the vite config, not a big problem if we do not want it exported from here though.